js/app.js

```diff
@@ -20,231 +20,254 @@ allAnswers = [];
 realAnswers = [];
 countries = ['canada', 'mexico', 'portugal'];
 
-//start questions
-alert('Hey, welcome to my site. My name is Austin Matteson but you may already know that. In fact, let\'s find out just how much you do know about me.');
-userName = prompt('But first, what is your name?');
-console.log('The user entered', userName);
-alert('Great name, ' + userName + ', let\'s begin the test!');
+function person() {
+  userName = prompt('But first, what is your name?');
+  console.log('The user entered', userName);
+  alert('Great name, ' + userName + ', let\'s begin the test!');
+}
 
 //question 1
-while (i < 1) {
-  currentAnswer = prompt('Was I born in Washington?');
-  currentAnswer = currentAnswer.toLowerCase();
-  if (currentAnswer === 'y') {
-    currentAnswer = 'yes';
-  } else if (currentAnswer === 'n') {
-    currentAnswer = 'no';
+function placeOfBirth() {
+  while (i < 1) {
```

Comment:
It is better practice to use a flag instead on incrementing and decrementing a global variable. That way, each function is self contained and does not depend on the value of the global to work properly.